code/src/index.css

```diff
@@ -1,13 +1,88 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(255, 144, 162);
+  font-family: 'Roboto', sans-serif;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.card {
+  margin: 20px;
+  box-shadow: 10px 10px 4px -4px black;
+  border: 2px solid black;  
+  padding: 0 20px;
+  background-color: white;
 }
+
+/*** MessageInput ***/
+
+textarea, input {
+  display: block;
+  margin-bottom: 20px;
+}
+
+.text-input {
+  padding: 5px;
+  width: calc(100% - 10px);
+  height: 50px;
+  background-color: rgb(243, 243, 243);
+  font-size: 12px;
+}
+
+.submit-button {
```

Comment:
Super tiny thinhg, but it could be nice with some :hover and/or :active effects on the button!